.gitattributes

```diff
@@ -1,3 +1,4 @@
+* text=auto eol=lf
```

Comment:
Shouldn't this be `* text=native` instead? Can you please give it a try?

Reply:
I don't see any `text=native` on git documentation => https://git-scm.com/docs/gitattributes
So, it's exotic (if working).

I have found a `core.eol=native` => https://git-scm.com/docs/git-config . 
In git attribute, may be `* text=auto eol=lf` be it's the opposite as the idea behind this PR: Force unix style evrywhere, anytime, because some tests use implicitly or explicitly `\n` on expected value. Today all text editor (even windows notepad) support LF style at least on read.

I don't currently have a Windows PC for few days, so I can't test your idea right now.

Reply:
We don't want any conversion to happen, the source should remain as committed, that's what we might want to describe.